src/Umbraco.Infrastructure/Mapping/UmbracoDefaultMapper.cs

```diff
@@ -25,6 +25,21 @@ public class UmbracoDefaultMapper : DefaultMapper
             };
         }
 
+        if (destType == typeof(DateOnly))
+        {
+            return value =>
+            {
+                return DateOnly.Parse(value.ToString()!);
+            };
+        }
+
+        if(destType == typeof(TimeOnly))
+        {
+            return value =>
+            {
+                return TimeOnly.Parse(value.ToString()!);
+            };
+        }
```

Comment:
I don't think this is necessary - see summary comment.

Reply:
Agreed @NguyenThuyLan, thanks - yes, I see the need for these when trying to read data.